docs/references/qa/CometBFT-QA-v1.md

```diff
@@ -0,0 +1,302 @@
+summarizes the values plotted in the figure. We can see the saturation point in the diagonal defined
+by `c=1,r=400` and `c=2,r=200`.
+
+| r    | c=1       | c=2       | c=4   |
+| ---: | --------: | --------: | ----: |
+| 200  | 17800     | **34600** | 50464 |
+| 400  | **31200** | 54706     | 49463 |
+| 800  | 51146     | 51917     | 41376 |
+| 1600 | 50889     | 47732     | 45530 |
+
+For comparison, this is the table obtained on the baseline version, with the same saturation point.
+
+| r    | c=1       | c=2       | c=4   |
+| ---: | --------: | --------: | ----: |
+| 200  | 17800     | **33259** | 33259 |
+| 400  | **35600** | 41565     | 41384 |
+| 800  | 36831     | 38686     | 40816 |
+| 1600 | 40600     | 45034     | 39830 |
+
+In conclusion, we chose `c=1,r=400` as the transaction load that we will use in the rest of QA
+process. This is the same value used in the previous QA tests.
+
+### With latency emulation
+
+For comparing the performance of `v1` with and without latency emulation, we run a new set of
+experiments with a different configurations of transaction loads: we use only one connection, and a
+transaction rate ranging from 100 to 1000 tx/s, in intervals of 100.
+
+![v1_saturation](imgs/v1/saturation/saturation_v1_LE.png) 
+
+These are the actual values which we use to generate the figure:
+
+| r    | v1 without LE | v1 with LE   | 
+| ---: | ----: | ----: |
+| 100  |  8900 |  8900 |
+| 200  | 17800 | 17800 |
+| 300  | 26053 | 26700 |
+| 400  | 28800 | 35600 |
+| 500  | 32513 | 34504 |
+| 600  | 30455 | 42169 |
+| 700  | 33077 | 38916 |
+| 800  | 32191 | 38004 |
+| 900  | 30688 | 34332 |
+| 1000 | 32395 | 36948 |
+
+## 200-nodes test
+
+This experiment consist in running the 200-nodes network, injecting a load of 400 tx/s (`c=1,r=400`)
+during 90 seconds, and collecting the metrics. The network composition is the same as used for
+finding the saturation point.
+
+For the experiments with latency emulation have set a duration of 180 seconds instead of the 90.
+
+### Latencies
+
+The following figures show the latencies of the experiment. Each dot represents a block: at which
+time it was created (x axis) and the average latency of its transactions (y axis).
+
+| v0.38 | v1 (without LE / with LE) 
+|:--------------:|:--------------:|
+| ![latency-1-400-v38](img38/200nodes/e_de676ecf-038e-443f-a26a-27915f29e312.png) | ![latency-1-400-v1](imgs/v1/200nodes/latencies/e_8e4e1e81-c171-4879-b86f-bce96ee2e861.png) 
+| | ![latency-1-400-v1-le](imgs/v1/200nodes_with_latency_emulation/latencies/e_8190e83a-9135-444b-92fb-4efaeaaf2b52.png)
+
+In both cases, most latencies are around or below 4 seconds. On `v0.38` there are peaks reaching 10
+seconds, while on `v1` (without LE) the only peak reaches 8 seconds. In general, the images are
+similar. From this small experiment we infer that the latencies measured on `v1` are not worse than
+those of the baseline. With latency emulation, the latencies are considerably higher, as expected.
```

Comment:
I guess indeed, with latency emulation, the system is trading latency for throughput.